.changes/unreleased/added-20250319-151718.yaml

```diff
@@ -0,0 +1,5 @@
+kind: added
+body: new `powerplatform_environment_wave` that allows to install newest wave update in an environment. Currently available only with `user authentication login`
```

Comment:
"Currently available only with `user authentication login" have we talked to API team about what the blocker is?

Reply:
I will, not that will change the current state of authentication to that endpoint.